pkg/trace/api/debug_server.go

```diff
@@ -0,0 +1,110 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
```

Comment:
```suggestion
// Copyright 2023-present Datadog, Inc.
```

Not sure it matters, but probably doesn't hurt.